### PR TITLE
Reports Fix

### PR DESCRIPTION
**Pull request by binaryoverload**
- [x] Completed PR - Has all features or fixes that it was meant to and isn't half done
- [x] Follows the style guide - Style guide found [here](https://github.com/FlareBot/FlareBot/blob/master/CONTRIBUTING.md)
- [x] Your code follows the same code style as the rest of the project
- [x] Tested changes

## What is added
Report Messages

## What is fixed
Reports

### CODE REVIEW

**Comment by WalshyDev** [comment]
Do reports need the guild wrapper for anything? Or could they just be saved in Cassandra and loaded when needed as planned before?

**Comment by binaryoverload** [comment]
I suppose we could :thinking:

**Comment by binaryoverload** [comment]
Kept the reports manager and added report messages @WalshyDev

File: src/main/java/stream/flarebot/flarebot/commands/general/ReportCommand.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package stream.flarebot.flarebot.commands.general;
 
+import net.dv8tion.jda.core.entities.ISnowflake;
```
Comment (WalshyDev) [inline src/main/java/stream/flarebot/flarebot/commands/general/ReportCommand.java]: Where is this used?

**Pull request merged**